packages/core/ios/Sources/DdSdk.swift

```diff
@@ -245,4 +274,17 @@ class RNDdSdk: NSObject {
             Datadog.verbosityLevel = nil
         }
     }
+    
+    func buildFrameTimeCallback(vitalsUpdateFrequency: Datadog.Configuration.VitalsFrequency) -> (Double) -> () {
+        func frameTimeCallback(frameTime: Double) {
+            if (vitalsUpdateFrequency != .never && frameTime > 0) {
+                Global.rum.updatePerformanceMetric(metric: .jsFrameTimeSeconds, value: frameTime)
+            }
+            if (frameTime > jsLongTaskThresholdInSeconds) {
+                // TODO: report long task
```

Comment:
I am not aware of a team TODO strategy, I think each repo follows its own.
I personally try to avoid them.

This one could actually be removed, it is for another task and should not be committed in this PR ideally.
I added it to show the team what my plan was for long task reporting and check with everyone if it made sense.

Reply:
We allow having TODO items, but with RUMM task ID. like `// TODO RUMM-1234`